test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/TxnVerbs.java

```diff
@@ -306,8 +325,11 @@ public static HapiContractCall contractCall(String contract) {
 		return new HapiContractCall(contract);
 	}
 
+	// The ternary operator enables the developer to pass either an ABI, or just the name of the function
 	public static HapiContractCall contractCall(String contract, String abi, Object... params) {
-		return new HapiContractCall(abi, contract, params);
+		return abi.charAt(0) == '{'
+				? new HapiContractCall(abi, contract, params)
+				: new HapiContractCall(getABIFor(FUNCTION, abi, contract), contract, params);
```

Comment:
Instead of passing as argument `String abi` that can represent either the function name we need to lookup from the file with `getABIFor` or the literal string representation of the ABI, we can move the static `getABIFor` to `com.hedera.services.bdd.suites.contract.Utils` and then call the `contractCall` method from a test suite like this:
- UtilVerbs.contractCall("ContractName", Utils.getABIFor(FUNCTION, "functionName", "ContractName"), args...);
- UtilVerbs.contractCall("ContractName", ContractResources.HW_MINT_CALL_ABI, args...);

This way we won't need the check for '{' at symbol index 0 in the string used as argument, which will work, but is not obvious and makes the code harder to understand.
I would also move the static `validateFileExists` and the enum for FunctionType out of `TxnVerbs`, probably to the `Utils` class, or maybe a new class for the enum. All other `TxnVerbs` methods return some type of Hapi operation and I think it makes sense to use the class only for that.

Reply:
Addressed after discussion with the reviewer.